### ISSUE TITLE
prepare my own data

### ISSUE THREAD

**Issue by strongteck** [skokec/segdec-net-jim2019#2]
Hello,
Could you please advise how to prepare my own data for training and testing using this code? I have a bunch of image files with defects or not, and I have also created the binary files with defects tagged.

Thank you,
Xinlong

**Comment by skokec** [comment]
You need to store the data as TFRecords using the tf.train.Example. The required features that need to be added into the tf.train.Example can be seen in [input_data/image_processing.py](https://github.com/skokec/segdec-net-jim2019/blob/2ebe7f5e4080f3c6f1d5117ff534583c20b97308/input_data/image_processing.py#L337). 

I have also just added the code that was used when creating KolektorSDD dataset where you can see all the details: [input_data/input_data_build_image_data_with_mask.py](https://github.com/skokec/segdec-net-jim2019/blob/master/input_data/input_data_build_image_data_with_mask.py).

**Comment by strongteck** [comment]
Will give a try.
Thank you!

**Comment by ShaneYS** [comment]
> 
> 
> Will give a try.
> Thank you!

hi, have you train your own data?
I also want  to train segdecnet on my own data( I have images and masks now) , could you tell me the steps to prepare data?
